.env.example

```
@@ -30,6 +30,8 @@ OAUTH_PROXY_TRUSTED_ORIGINS='http://localhost:3712,https://your-preview-origin-p
 # @see https://www.better-auth.com/docs/authentication/github
 AUTH_GITHUB_ID=''
 AUTH_GITHUB_SECRET=''
+# Optional: raises rate limits for public GitHub skill imports.
+GITHUB_IMPORT_TOKEN=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)